src/server/routes/monthData.js

```diff
@@ -54,14 +99,35 @@ function selectClientMonthData(req, res) {
       console.error(`Status Code: ${error.statusCode}, Error Message: ${error.code}`);
 
       res.json({
-        statusCode: error.statusCode,
+        statusCode: error.statusCode || 500,
         errorMessage: error.code,
       });
     });
 }
 
+/**
+ * selectClientMonthDataPost(req, res, next)
+ * Handles the requests from the form submit button (when no JS).
+ * It redirects to the route to execute the function for server side requesting.
+ */
+function selectClientMonthDataPost(req, res) {
+  const season = (req.body.season) ? `${req.body.season}-01` : '';
```

Comment:
I don't like how `-01` is hardcoded here. It seems likely to fail if the API is updated.

Reply:
Because from the URL, the season/month's convention always has `[year]-[month]-01` in the end. However, the API only need `[year]-[month]`. I think `-01` is redundant, but we already have it from beginning.

Reply:
I'm not sure if @gkallenberg finished working on the API yet but the previous API endpoints had `[year]-[month]-01` as the convention. If we move away from that convention, then yes it is redundant and we don't need it. But also remember that users will have URLs that are in the form of `[year]-[month]-01`.

Reply:
So far if you check swagger it is `[year]-[month]`, so I just follow that convention.

Reply:
Greg posted that it's now `[year]-[month]-01` in Slack a few hours ago. Let's stick to that.

Reply:
Cool will update it